apps/api/src/app/workflows-v2/workflow.controller.e2e.ts

```diff
@@ -1062,10 +1092,11 @@ function buildEmailStep(): StepCreateDto {
     type: StepTypeEnum.EMAIL,
   };
 }
-function buildDigestStep(): StepCreateDto {
+function buildDigestStep(overrides: Partial<StepCreateDto> = {}): StepCreateDto {
```

Comment:
control values should not be updated from here anymore, we have a patch step, use it, I'm going to delete this functionality soon